### PR TITLE
To Prod: GET member API with family_info

### PR DESCRIPTION
**Pull request by tkmktkhr**
## Issues Link

https://github.com/t-k-collab/rsapi/issues/15
https://github.com/t-k-collab/rsapi/issues/16

## Other

### CODE REVIEW

File: src/entities/member.rs [hunk 1]
```diff
@@ -1,32 +1,41 @@
...
 pub struct MemberEntity {
-    pub member_id: u16,
+    pub member_id: i32,
     pub family_name: String,
     pub middle_name: String,
     pub first_name: String,
-    pub pass_code: String,
-    pub created: String, // TODO date type. time or chrono crate. // TODO this is only for model.
-    pub updated: String, // TODO date type. time or chrono crate. // TODO this is only for model.
+    pub date_of_birth: String,
```
Comment (tkmktkhr) [inline src/entities/member.rs:10]: might mistake??

File: src/interfaces/controllers/members/find_member.rs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    ) -> Result<(StatusCode, FindMemberOutputData), (StatusCode, CustomError)> {
+        let input_data = FindMemberInputData::new(member_id);
+        let result = use_case.find_member(input_data).await;
+
+        match result {
+            Ok(result) => match result {
+                None => Err::<_, (StatusCode, CustomError)>((
+                    StatusCode::NOT_FOUND,
+                    CustomError::NotFoundError {
+                        msg: "not found".to_string(),
```
Comment (tkmktkhr) [inline src/interfaces/controllers/members/find_member.rs:25]: brush up here later.

File: src/use_cases/members/find_member.rs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        let result = self.repo.find_with_family_info(input_data).await;
+
+        match result {
+            Ok(None) => {
+                println!("get response None from repository find_with_family_info.");
+                Ok(None)
+            } // here will not be passed?? If there are no members, it will return empty array [].
+            Ok(Some(res)) => {
+                let families = if res.is_empty() {
+                    return Ok(None);
```
Comment (tkmktkhr) [inline src/use_cases/members/find_member.rs:53]: maybe not happen

**Review by Akomea — APPROVED** [review]
Looking good, I'll integrate the changes to the frontend and ask more questions

**Pull request merged**